ops/README.md

```diff
@@ -0,0 +1,72 @@
+# docker-compose
+
+The docker-compose project runs a local optimism stack.
+
+## prerequisites
+
+- docker
+- docker-compose
+- make
+
+## Starting and stopping the project
+
+The base `docker-compose.yml` file will start the required components for a full stack.
+
+Supplementing the base configuration is an additional metric enabling file, `docker-compose-metrics.yml`. Adding this configuration to the stack will enable metric emission for l2geth and start grafana (for metrics visualisation) and influxdb (for metric collection) instances.
+
+The base stack can be started and stopped with a command like this (there is no need to specify the default docker-compose.yml)
+```
+docker-compose \
+    up --build --detach
+```
+
+To start the stack with monitoring enabled, just add the metric composition file.
+```
+docker-compose \
+    -f docker-compose.yml \
+    -f docker-compose-metrics.yml \
+    up --build --detach
+```
+
+A Makefile has been provided for convience. The following targets are available.
+- make up
+- make down
+- make up-metrics
+- make down-metrics
+
+## Authentication
+
+Influxdb has authentication disabled.
+
+Grafana requires a login. The defaults are:
+```
+user: admin
+password: optimism
+```
+
+## Data persistance
+
+Grafana data is not currently saved. Any modifications or additions will be lost on container restart.
+
+InfluxDB is persisting data to a Docker volume.
+
+**Stopping the project removing the containers will not clear this volume**
+
+To remove the influxdb data, run a command like
+```
+docker volume rm ops_influxdeb_data
+```
+Or, after stopiing the project, clean up all unused containers and volumes.  
+
+**This step remove way more then just this project, please make sure you know what you're doing**
+```
+docker system prune --volumes
```

Comment:
Maybe we should just remove this suggestion? I can see somebody running it by accident..

Reply:
Yeah, I wasn't 💯 on this, heh. Removing.